app/test_split_integer.py

```diff
@@ -2,20 +2,23 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    parts = split_integer(8, 1)
+    assert sum(parts) == 8
```

Comment:
u don't need to create a separated variable in this case
just do it in one line